scripts/webframeworks-deploy-tests/tests.ts

```diff
@@ -39,13 +40,17 @@ describe("webframeworks deploy", function (this) {
     // This is not an empty block.
   });
 
-  it("deploys functions with runtime options", async () => {
+  it("logs reasons for backend", async () => {
     process.env.FIREBASE_CLI_EXPERIMENTS = "webframeworks";
 
     const result = await setOptsAndDeploy();
 
-    expect(result.stdout, "deploy result").to.match(/file upload complete/);
-    expect(result.stdout, "deploy result").to.match(/found 20 files/);
```

Comment:
can you glob the `.firebase/SITE/hosting` directory and count / compare the files?